tasks/dnf.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Configure dnf-automatic
+  block:
+    - name: Configure dnf-automatic to only consider security updates
```

Comment:
Would a loop make sense considering the overlap in all but the `option` and `value` arguments?

Reply:
Please see commit [6a503c7](https://github.com/cisagov/ansible-role-automated-security-updates/pull/38/commits/6a503c75b9cfdd650ecca49be0e84460a83ff476).